src/ntimporters/trello/importer.py

```diff
@@ -0,0 +1,344 @@
+    }
+    if (
+        len(trello_tags := trello_client.tags(project.get("id"))) + len(nt_tags)
+        > limits.get("tags")
+        > -1
+    ):
+        raise ImportException("LIMIT tags")
+    for tag in trello_tags:
+        if (tag_name := tag.get("name")) not in nt_tags and (
+            nt_tag := nt_api_tags.post_tag(
+                strip_readonly(
+                    models.Tag(
+                        models.Id16ReadOnly(id16()),
+                        models.Name(tag_name),
+                        color=_map_color(tag.get("color")),
+                    )
+                )
+            )
+        ):
+            nt_tags[tag_name] = str(nt_tag.id)
+    return nt_tags
+
+
+def _import_tags(nt_client, nt_task_id: str, task: dict, tags_mapping):
+    """Assign tags to task"""
+    nt_api_tag_assignments = apis.TagAssignmentsApi(nt_client)
+    for tag in task.get("labels"):
+        if nt_tag_id := tags_mapping.get(tag.get("name")):
+            nt_api_tag_assignments.post_tag_assignment(
+                strip_readonly(
+                    models.TagAssignment(
+                        id=models.Id16ReadOnly(id16()),
+                        tag_id=models.Id16(nt_tag_id),
+                        task_id=models.Id16(nt_task_id),
+                    )
+                )
+            )
+
+
+def nt_members_by_email(nt_client) -> Tuple[dict, str]:
+    """Map NT emails to member ids"""
+    nt_members = {
+        str(elt.user_id): str(elt.id) for elt in apis.TeamMembersApi(nt_client).get_team_members()
+    }
+    mapping = {}
+    current_user_id = None
+    for user in apis.UsersApi(nt_client).get_users():
+        if hasattr(user, "email"):
+            email = user.email
+        elif hasattr(user, "invitation_email"):
+            email = user.invitation_email
+        if bool(user.is_me):
+            current_user_id = str(user.id)
+        mapping[str(email)] = nt_members.get(str(user.id))
+    return mapping, nt_members.get(current_user_id)
+
+
+def _import_comments(nt_client, trello_client, nt_task_id: str, tr_task_id: str):
+    """Import task-related comments"""
+    nt_api_comments = apis.CommentsApi(nt_client)
+    for comment in sorted(
+        trello_client.comments(tr_task_id), key=lambda elt: isoparse(elt.get("date")).timestamp()
+    ):
+        nt_api_comments.post_comment(
+            strip_readonly(
+                models.Comment(
+                    id=models.Id16ReadOnly(id16()),
+                    body=comment.get("text"),
+                    task_id=models.Id16(nt_task_id),
+                    author_id=models.Id16ReadOnly(id16()),
+                    created_at=models.TimestampReadOnly(1),
+                    extra="",
+                )
+            )
+        )
+
+
+def _map_color(trello_color: Optional[str]) -> models.Color:
+    """Maps Trello color onto Nozbe Teams color"""
+    return models.Color(trello_color if trello_color in COLORS else random.choice(COLORS))  # nosec
```

Comment:
how about using `models.Color.allowed_values`, or wrapping `models.Color(trello_color)` in try/catch clause, instead of copy-pasting list of allowed colors?

Reply:
not bad idea